packages/contracts/scripts/deploy/manager.ts

```diff
@@ -1,102 +1,119 @@
+import stakingFunc, { optionDefinitions as stakingOptions } from "./dollar/Staking";
+import stakingFormulasFunc, { optionDefinitions as stakingFormulasOptions } from "./dollar/StakingFormulas";
+import stakingShareFunc, { optionDefinitions as stakingShareOptions } from "./dollar/StakingShare";
+import ubiquityChefFunc, { optionDefinitions as ubiquityChefOptions } from "./dollar/UbiquityChef";
+import ubiquityFormulasFunc, { optionDefinitions as ubiquityFormulasOptions } from "./dollar/UbiquityFormulas";
+import sushiSwapPoolFunc, { optionDefinitions as sushiSwapPoolOptions } from "./dollar/SushiSwapPool";
+/* 
 import uARForDollarsCalculatorFunc, { optionDefinitions as uARForDollarsCalculatorOptions } from "./dollar/UARForDollarsCalculator";
 import ubiquityAlgorithmicDollarFunc, { optionDefinitions as ubiquityAlgorithmDollarOptions } from "./dollar/UbiquityAlgorithmicDollar";
 import ubiquityAutoRedeemFunc, { optionDefinitions as ubiquityAutoRedeemOptions } from "./dollar/UbiquityAutoRedeem";
-import ubiquityFormulaFunc, { optionDefinitions as ubiquityFormulaOptions } from "./dollar/UbiquityFormulas";
-import yieldProxyFunc, { optionDefinitions as yieldProxyOptions } from "./dollar/YieldProxy";
+ */
 import ubiquiStickFunc, { optionDefinitions as ubiquiStickOptions } from "./ubiquistick/UbiquiStick";
 import ubiquiStickSaleFunc, { optionDefinitions as ubiquiStickSaleOptions } from "./ubiquistick/UbiquiStickSale";
 import uARFunc, { optionDefinitions as uAROptions } from "./ubiquistick/UAR";
 import lpFunc, { optionDefinitions as lpOptions } from "./ubiquistick/LP";
 import simpleBondFunc, { optionDefinitions as simpleBondOptions } from "./ubiquistick/SimpleBond";
 
 export const DEPLOY_FUNCS: Record<string, { handler: DeployFuncCallback; options: OptionDefinition[] }> = {
-  Bonding: {
-    handler: bondingFunc,
-    options: bondingOptions,
+  DirectGovernanceFarmer: {
+    handler: directGovernanceFarmerFunc,
+    options: directGovernanceFarmerOptions,
+  },
+  Erc20Ubiquity: {
+    handler: erc20UbiquityFunc,
+    options: erc20UbiquityOptions,
+  },
+  ERC1155Ubiquity: {
+    handler: erc1155UbiquityFunc,
+    options: erc1155UbiquityOptions,
   },
-  UbiquityAlgorithmicDollarManager: {
-    handler: uAdManagerFunc,
-    options: uadManagerOptions,
+  Staking: {
+    handler: stakingFunc,
+    options: stakingOptions,
   },
-  BondingShare: {
-    handler: bondingShareFunc,
-    options: bondingShareOptions,
+  StakingFormulas: {
+    handler: stakingFormulasFunc,
+    options: stakingFormulasOptions,
   },
-  BondingShareV2: {
-    handler: bondingShareV2Func,
-    options: bondingShareV2Options,
+  StakingShare: {
+    handler: stakingShareFunc,
+    options: stakingShareOptions,
   },
-  CouponsForDollarsCalculator: {
-    handler: couponsForDollarsCalculatorFunc,
-    options: couponsForDollarsCalculatorOptions,
+  ubiquityChef: {
```

Comment:
Why is this the only one not capitalized?

Reply:
```suggestion
  UbiquityChef: {
```

Reply:
> 

it is done but it seems that for GH it is still requested